includes/abstracts/abstract-wc-order.php

```diff
@@ -145,6 +145,39 @@ public function remove_order_items( $type = null ) {
 	}
 
 	/**
+	 * Returns a list of all payment tokens associated with the current order
+	 *
+	 * @since 2.6
+	 * @return array An array of payment token objects
+	 */
+	public function get_payment_tokens() {
+		return WC_Payment_Tokens::get_order_tokens( $this->id );
+	}
+
+	/**
+	 * Add a payment token to an order
+	 *
+	 * @since 2.6
+	 * @param  WC_Payment_Token   $token     Payment token object
+	 * @return boolean True if the token was added, false if not
+	 */
+	public function add_payment_token( $token ) {
+		if ( empty( $token ) || ! ( $token instanceof WC_Payment_Token ) ) {
+			return false;
+		}
+
+		$token_ids = get_post_meta( $this->id, '_wc_payment_tokens', true );
```

Comment:
I agree with @thenbrent, this can be `_payment_tokens` since it's core order meta